packages/rs-drive-proof-verifier/src/types.rs

```diff
@@ -614,3 +617,162 @@ pub struct ProposerBlockCountByRange(pub u64);
 #[derive(Debug)]
 #[cfg_attr(feature = "mocks", derive(serde::Serialize, serde::Deserialize))]
 pub struct ProposerBlockCountById(pub u64);
+
+#[derive(Debug, Clone)]
+#[cfg_attr(
+    feature = "mocks",
+    derive(Encode, Decode, PlatformSerialize, PlatformDeserialize),
+    platform_serialize(unversioned)
+)]
+/// Status of a network node.
+///
+/// Some fields may be `None` if the information is not available, eg. when the node is not fully functional.
+pub struct EvonodeStatus {
```

Comment:
Let's have it nested as proto otherwise it's too messy

Reply:
done